--- a/specifications/chef.gemspec	2012-05-13 20:19:29.976755585 -0400
+++ b/specifications/chef.gemspec	2012-05-13 20:24:53.812753711 -0400
@@ -29,7 +29,7 @@
       s.add_runtime_dependency(%q<ohai>, [">= 0.6.0"])
       s.add_runtime_dependency(%q<rest-client>, ["< 1.7.0", ">= 1.0.4"])
       s.add_runtime_dependency(%q<bunny>, [">= 0.6.0"])
-      s.add_runtime_dependency(%q<json>, ["<= 1.6.1", ">= 1.4.4"])
+      s.add_runtime_dependency(%q<json>, ["<= 1.6.5", ">= 1.4.4"])
       s.add_runtime_dependency(%q<yajl-ruby>, [">= 1.1.0"])
       s.add_runtime_dependency(%q<treetop>, ["~> 1.4.9"])
       s.add_runtime_dependency(%q<net-ssh>, ["~> 2.2.2"])
@@ -56,7 +56,7 @@
       s.add_dependency(%q<ohai>, [">= 0.6.0"])
       s.add_dependency(%q<rest-client>, ["< 1.7.0", ">= 1.0.4"])
       s.add_dependency(%q<bunny>, [">= 0.6.0"])
-      s.add_dependency(%q<json>, ["<= 1.6.1", ">= 1.4.4"])
+      s.add_dependency(%q<json>, ["<= 1.6.5", ">= 1.4.4"])
       s.add_dependency(%q<yajl-ruby>, [">= 1.1.0"])
       s.add_dependency(%q<treetop>, ["~> 1.4.9"])
       s.add_dependency(%q<net-ssh>, ["~> 2.2.2"])
@@ -84,7 +84,7 @@
     s.add_dependency(%q<ohai>, [">= 0.6.0"])
     s.add_dependency(%q<rest-client>, ["< 1.7.0", ">= 1.0.4"])
     s.add_dependency(%q<bunny>, [">= 0.6.0"])
-    s.add_dependency(%q<json>, ["<= 1.6.1", ">= 1.4.4"])
+    s.add_dependency(%q<json>, ["<= 1.6.5", ">= 1.4.4"])
     s.add_dependency(%q<yajl-ruby>, [">= 1.1.0"])
     s.add_dependency(%q<treetop>, ["~> 1.4.9"])
     s.add_dependency(%q<net-ssh>, ["~> 2.2.2"])
